src/ServiceProvider.php

```diff
@@ -30,7 +30,10 @@ public function register(): void
         // Always available, even when inactive
         $this->registerFacades();
 
-        if (false === config('elastic-apm-laravel.active')) {
+        if (
+            false === config('elastic-apm-laravel.active')
+            || ('cli' === php_sapi_name() && false === config('elastic-apm-laravel.cli.active'))
```

Comment:
If I'm reading this conditional correctly, this will never register the Agent, middlewares, and collectors when `elastic-apm-laravel.active` is `true` but we are executing via CLI and `elastic-apm-laravel.cli.active` is false.

Reply:
If I read your comment correctly, that's right.

If `.active` is `false`, it will not register, regardless of request or `cli.active`.

If `.active` is `true` and we are on HTTP, it will register.

If `.active` is `true`, we are on CLI, and `cli.active` is false, it will not register.

If `.active` is `true`, we are on CLI, and `cli.active` is true, it will register.